### DISCUSSION TITLE
Blocking the /home command on the claim area

### DISCUSSION THREAD

**Discussion by Xenovac** [GriefPrevention/GriefPrevention#2001] [Ideas]
I want an addon that restrict players to teleport to areas that has been claimed by other players.

I would be very happy if you could do this favor.

**Comment by Jikoo** [comment]
You already can block commands inside claim area by using the node `GriefPrevention.Claims.CommandsRequiringAccessTrust`. This is a semicolon-separated list that defaults to `/sethome`. You should include any other commands you wish to block inside claims and their aliases.
